### PR TITLE
Fix indentation in docs

### PR DESCRIPTION
**Pull request by philipkiely-baseten**

### CODE REVIEW

File: docs/develop/processing.md [hunk 1]
```diff
@@ -59,16 +61,18 @@ import requests
...
-    request = requests.get(model_input)
-    with tempfile.NamedTemporaryFile() as f:
-        f.write(request.content)
-        f.seek(0)
-        input_image = tf.image.decode_png(tf.io.read_file(f.name))
-    preprocessed_image = tf.keras.applications.resnet_v2.preprocess_input(
-        tf.image.resize([input_image], (224, 224))
-    )
-    return np.array(preprocessed_image)
+class Model:
```
Comment (bolasim) [inline docs/develop/processing.md:64]: this also needs a predict to make sense I think. Or you need a `...` under `class Model:`

File: docs/develop/processing.md [hunk 2]
```diff
@@ -82,17 +86,19 @@ Here is a post-processing function from the same [TensorFlow example](../create/
...
-        "https://storage.googleapis.com/download.tensorflow.org/data/ImageNetLabels.txt"
-    ).text.split("\n")
-    class_probabilities = softmax(class_predictions)
-    top_probability_indices = class_probabilities.argsort()[::-1][:5].tolist()
-    return {
-        LABELS[index]: 100 * class_probabilities[index].round(3)
-        for index in top_probability_indices
-    }
+class Model:
+
```
Comment (bolasim) [inline docs/develop/processing.md:90]: same as above

File: docs/develop/secrets.md [hunk 3]
```diff
@@ -39,23 +39,20 @@ secrets:
...
-self.s3_config = (
-    {
-        "aws_access_key_id": secrets["gfpgan_aws_access_key_id"],
-        "aws_secret_access_key": secrets["gfpgan_aws_secret_access_key"],
-        "aws_region": secrets["gfpgan_aws_region"],
-    }
-)
-self.s3_bucket = (secrets["gfpgan_aws_bucket"])
+class Model:
+
```
Comment (bolasim) [inline docs/develop/secrets.md:43]: same

**Review by bolasim — APPROVED** [review]
find to merge for now, just thinking about how to do these code spinnets in a way that's complete without being too verbose. It's a hard balance. I think we're getting close.

File: docs/develop/processing.md (hunk 1 above)
Comment (philipkiely-baseten) [reply docs/develop/processing.md:64]: Will add predict and load stubs so code doesn't show errors when copied like it would for ...

**Pull request merged**